features/login.feature

```diff
@@ -0,0 +1,63 @@
+Feature: Login behavior
+@QA
+Scenario: Ensure login shows proper error messages
+    Then I install the ccz app at "test_list_search.ccz"
+
+    # normal login
+    Then I login with username "user_with_no_data" and password "123"
+    # TODO: consider counting the home screen buttons
+    Then I logout
+
+    # login in portrait mode
```

Comment:
Can you change this comment to say "landscape mode". Also curious why the step definition we have defined doesn't work in landscape?